src/index.ts

```diff
@@ -7,3 +7,10 @@ ethers.provider.on("block", async () => {
   const reserves = await pair.getReserves();
   console.log(`${reserves[0].toString()} ----- ${reserves[1].toString()}`);
 });
+
+ethers.provider.on("block", async () => {
```

Comment:
Hey @Scapestudio , I'd like to see both the uniswap and sushiswap reserve calls inside the same "block" event handler. Can you consolidate it so that instead of 2 event listeners, we only have 1?